components/gitpod-protocol/src/teams-projects-protocol.ts

```diff
@@ -14,6 +14,7 @@ export interface ProjectConfig {
 }
 
 export interface ProjectSettings {
+    enablePrebuilds?: boolean;
     useIncrementalPrebuilds?: boolean;
```

Comment:
The backwards compatibility logic that `undefined` actually means `enabled` is very surprising. What are you concerns with usin the inverse logic internally? I.e. `prebuildsDisabled`?

Reply:
If you do a data migration, then I'd love to discuss if a shape for prebuilds as suggested by @geropl would make sense. Also would be good to name the property `enabled` instead of `enable` as the later sounds like an action and not like a state.